Corona-Warn-App/src/main/res/values-en/strings.xml

```diff
@@ -185,7 +185,7 @@
     <!-- XTXT: main, explains faq on card -->
     <string name="main_about_body">"Here you can find answers to frequently asked questions about the Corona-Warn-App. You will be forwarded to an external website."</string>
     <!-- NOTR -->
-    <string name="main_about_link">"https://www.bundesregierung.de/corona-warn-app-faq-englisch"</string>
+    <string name="main_about_link">"https://www.bundesregierung.de/corona-warn-app-faq"</string>
```

Comment:
@kaluju that's wrong for my understanding, my should use the english FAQ link or have I missed something?